pytest_mozwebqa/selenium_client.py

```diff
@@ -139,6 +142,16 @@ def start_client(self):
             capabilities.update(webdriver.DesiredCapabilities.OPERA)
             self.selenium = webdriver.Opera(executable_path=self.opera_path,
                                             desired_capabilities=capabilities)
+        elif self.driver.upper() == 'BROWSERSTACK':
```

Comment:
I notice that in other code you are using `lower()` on the driver name (e.g., [1]), which I realize is needed because the module name is in lower case, but here you are using `upper()` in order to normalize the case. Is there a reason you chose to use `upper()` here instead of `lower()`? It does make the code stand out, to see all those CAPS, which may be why, but it also is a bit ugly/looks non-pythonic, which is why it caught my attention.

I guess this is really a pretty minor nit, but I was wondering why you chose to code it this way.

[1] https://github.com/mozilla/pytest-mozwebqa/pull/152/files#diff-d96b35188b948d8041ba00283c6631b3R27